library/lua/gui/widgets.lua

```diff
@@ -657,6 +657,12 @@ function EditField:onRenderBody(dc)
     dc:string((' '):rep(dc.clip_x2 - dc.x))
 end
 
+function EditField:insert(text)
```

Comment:
idea: might be nice to have an optional `cursor` param someday

Reply:
I was thinking that as I wrote the docs, but then I looked at what it would take to bounds check the parameter and I thought..maybe I'll leave that refactoring for another day : p